mcc/src/org/labkey/mcc/MccModule.java

```diff
@@ -81,4 +82,23 @@ public void doStartup(ModuleContext moduleContext)
     {
         return Collections.singleton(MccSchema.NAME);
     }
+
+    private void registerEHRResources()
+    {
+        EHRService.get().registerModule(this);
+        //EHRService.get().registerTableCustomizer(this, ONPRC_EHRCustomizer.class);
+
+        //Resource r = getModuleResource("/scripts/mcc/mcc_triggers.js");
+        //assert r != null;
+        //EHRService.get().registerTriggerScript(this, r);
+
+        //EHRService.get().registerClientDependency(ClientDependency.supplierFromPath("Ext4"), this);
+        //EHRService.get().registerClientDependency(ClientDependency.supplierFromPath("onprc_ehr/panel/BloodSummaryPanel.js"), this);
```

Comment:
Is it safe to delete these commented lines?

Reply:
yeah, this module is very much in-progress. some of it was a placeholder for me to remember what i need to do